euphoria-core/src/main/java/cz/seznam/euphoria/core/client/dataset/Datasets.java

```diff
@@ -39,9 +42,9 @@
    * @return a dataset representing the output of the given operator
    */
   public static <IN, OUT> Dataset<OUT> createOutputFor(
-      Flow flow, Dataset<IN> input, Operator<IN, OUT> op) {
+      Flow flow, Dataset<IN> input, Operator<IN, OUT> op, Set<OutputHint> outputHints) {
```

Comment:
If we set output hints on operator, there is no need for additional parameter here. On Beam branch, I already further simplified this method, because the parameter `flow` is redundant, because it can be retrieved from both `input` and `op`.

Reply:
Hints are now on operator